track_analyzing/track_analyzer/cmd_table.cpp

```diff
@@ -41,6 +47,9 @@ using namespace track_analyzing;
 
 namespace
 {
+uint16_t constexpr kWalkMaxspeedKMpHValue = 10;
+uint16_t constexpr kNoneMaxspeedKMpHValue = 200;
```

Comment:
Why not to use 
```c++
uint16_t constexpr kInvalidSpeed = std::numeric_limits<uint16_t>::max();
uint16_t constexpr kNoneMaxSpeed = std::numeric_limits<uint16_t>::max() - 1;
uint16_t constexpr kWalkMaxSpeed = std::numeric_limits<uint16_t>::max() - 2;
```
?

Reply:
```kNoneMaxSpeed``` and ```kWalkMaxSpeed``` - not the things I need to. I need some meaningful speed to show in the resulting csv file. Also ```kNoneMaxspeedKMpHValue``` is being used as a upper bound for any speed in the resulting csv. 65534 and 65533 make no sense during analysis and would be another weird looking constant in the resulting file.